README.md

```diff
@@ -113,3 +113,20 @@ user=<user>
 branch=<branch>
 bin/ship -y -a log-iss -b $branch -u $user -i splunk-indexer -d 20 -n 5 #log-iss runs on the indexers
 ```
+
+### updating configs
+
+If you only need to update a configuration, you can do the following:
+```
+# update the config
+ion-client config:set -i splunk-indexer <ENV_VAR>=<new_value>
+
+# push the updated configs
+ic ssh deploymaster
+su deploymaster
+user=<user>
+bin/update-keys -u $user -i splunk-indexer
+
+# restart log-iss
+bin/run -u $user -i splunk-indexer -c '/usr/local/bin/app-restart log-iss'
+```
```

Comment:
@jmervine we should avoid custom commands if we can. Deploymaster already has builtin ways to do this. I'd suggest using: 

`bin/deploy -u$ user -i splunk-indexer -k -c log-iss`.

The -k will cause keys to get updated. The deploy will cause the service to restart.

Reply:
I'm looking for a away to update configs without doing a deploy. Only a way of updating the config and restarting the services.

Reply:
A deploy will be a no-op here though, and it will restart the service. So it boils down to exactly what you're doing - it updates configs and it restarts log-iss. Except that it doesn't have custom code.

A "deploy" is what you actually *want to do* :).